lib/datasource/github-releases/index.ts

```diff
@@ -38,6 +40,11 @@ export async function getReleases({
   if (cachedResult) {
     return cachedResult;
   }
+  let depHost = registryUrl;
+  if (ensureTrailingSlash(depHost) === 'https://github.com/') {
+    depHost = null;
```

Comment:
I don't understand this. Isn't it written back again in the subsequent code block?

Reply:
I think the only reason for this is to let `apiBaseUrl` point to _api.github.com_ in case github is passed as `registryUrl`, isn't it?

Reply:
yes, thats the cause. Maybe i can optimize /cleanup it some more, but first need to check, why some tests are failing now.

Any hints are helpful. 🙃

Reply:
Maybe an issue with the cache? It doesn't care about the `registryUrl` at all, the namespace is always set to **datasource-github-releases**.

https://github.com/renovatebot/renovate/blob/eca09cb1fd6bf17ddfc27d4c79c65191155e6968/lib/datasource/github-releases/index.ts#L35-L42

Reply:
I would just turn it off for a second and run the tests again to see if it helps.

Reply:
caching is turned of by default on tests, but iv'e fixed the cache keys now